build/jobs/provision-deploy.yml

```diff
@@ -88,3 +89,21 @@ jobs:
         Write-Host "Setting Key Vault access"
         Set-AzKeyVaultAccessPolicy -VaultName $webAppName -ObjectId 4d4d503d-9ca8-462e-9e18-b35fc8b5285b -PermissionsToSecrets list,get
 
+        $healthCheckUrl = "https://${{ parameters.webAppName }}.azurewebsites.net/health/check"
```

Comment:
Would it be useful to pull this health check into a separate job so that we can call this from redeploy-webapp.yml as well?